lib/vector-api-client/graphql/schema.json

```diff
@@ -1,45 +1,106 @@
 {
   "data": {
     "__schema": {
+      "directives": [
+        {
+          "args": [],
+          "description": "Directs the executor to query only when the field exists.",
+          "locations": [
+            "FIELD"
+          ],
+          "name": "ifdef"
+        },
+        {
+          "args": [
+            {
+              "defaultValue": null,
+              "description": "Included when true.",
+              "name": "if",
+              "type": {
+                "kind": "NON_NULL",
+                "name": null,
+                "ofType": {
+                  "kind": "SCALAR",
+                  "name": "Boolean",
+                  "ofType": null
+                }
+              }
+            }
+          ],
+          "description": "Directs the executor to include this field or fragment only when the `if` argument is true.",
+          "locations": [
+            "FIELD",
+            "FRAGMENT_SPREAD",
+            "INLINE_FRAGMENT"
+          ],
+          "name": "include"
+        },
+        {
+          "args": [
+            {
+              "defaultValue": null,
+              "description": "Skipped when true.",
+              "name": "if",
+              "type": {
+                "kind": "NON_NULL",
+                "name": null,
+                "ofType": {
+                  "kind": "SCALAR",
+                  "name": "Boolean",
+                  "ofType": null
+                }
+              }
+            }
+          ],
+          "description": "Directs the executor to skip this field or fragment when the `if` argument is true.",
+          "locations": [
+            "FIELD",
+            "FRAGMENT_SPREAD",
+            "INLINE_FRAGMENT"
+          ],
+          "name": "skip"
+        }
+      ],
+      "mutationType": null,
       "queryType": {
         "name": "Query"
       },
-      "mutationType": null,
```

Comment:
I wonder if the request changed, or the order for the generated fields is unstable? If we're using hashmap internally somehwere it might very well be that the generated schema fields aren't guaranteed to serialize in the predefined order. If so, we have to solve it, so regenerations produce consistent results.

Reply:
We bumped to async-graphql 2.0.0 [this morning](https://github.com/timberio/vector/pull/4537), so it might have tweaked how this file is being generated.

If the order winds up being unstable between `cargo run --bin graphql-schema` runs, I'll file an issue at async-graphql, but I haven't noticed that previously.